api/src/certification/enrolment/domain/usecases/get-enrolled-candidates-in-session.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @typedef {import("./index.js").EnrolledCandidateRepository} EnrolledCandidateRepository
```

Comment:
Je propose d'ajouter le lien, cela aidera les outils de dev a lire le `@returns`

```suggestion
 * @typedef {import("./index.js").EnrolledCandidateRepository} EnrolledCandidateRepository
 * @typedef {import("../read-models/EnrolledCandidate.js").EnrolledCandidate} EnrolledCandidate
```